### DISCUSSION TITLE
Ability in settings to move url bar to bottom or other side of screen

### DISCUSSION THREAD

**Discussion by CDLProduction** [zen-browser/desktop#3127] [Ideas]
I enjoy feature that browser could hide all bars and tool bars and leave clear web page, but I missing the feature to move URL bar to bottom side of screen, for me its easier to use in that way, and I think its more efficient and less time consuming, you don't have to drag mouse every time to top, as 80-90% of time mouse are in the center of screen and in down part of screen, mouse going up only in few cases.
Will be great to add this possibility to settings, for users like me.

**Comment by TheSethRose** [comment]
Would love to be able to hide the default URL bar completely, so that way it ONLY shows the floating URL bar.
